dashboard/test/controllers/api/v1/sections_students_controller_test.rb

```diff
@@ -37,4 +37,27 @@ class Api::V1::SectionsStudentsControllerTest < ActionController::TestCase
     ].to_json
     assert_equal expected_summary, @response.body
   end
+
+  test 'teacher can update gender, name and age info for their student' do
+    sign_in @teacher
+    put :update, params: {section_id: @section.id, id: @student.id, gender: 'f', age: 9, name: 'testname'}
+    assert_response :success
+    assert_equal 'f', JSON.parse(@response.body)['gender']
+    assert_equal 9, JSON.parse(@response.body)['age']
+    assert_equal 'testname', JSON.parse(@response.body)['age']
```

Comment:
I hope this doesn't pass! The response body `age` should not be `testname` :p

It's probably worth also validating that the the `student` was actually updated in the DB, at least one field:
```ruby
assert_equal 'testname', @student.reload.name
```